domovoi.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+import argparse
+import logging
+import os
+import shlex
+import sys
+import time
+
+import settings
+
+from solar_car_process import SolarCarProcess
+
+
+class Domovoi:
+        
+    def run(self, processes_file):
+        with open(processes_file) as file:
+            solar_car_processes = [SolarCarProcess(shlex.split(path)) for path in file.read().splitlines()]
```

Comment:
Making this file parsing a separate function in `Domovoi` increases the testability.